src/templates/components/CommunityTemplateReadme.tsx

```diff
@@ -41,3 +43,25 @@ export class CommunityTemplateReadme extends PureComponent<Props> {
     )
   }
 }
+
+const mstp = (state: AppState, props: any) => {
+  const templateDetails = getTemplateNameFromUrl(props.url)
+  return {
+    directory: templateDetails.directory,
+    name: templateDetails.name,
+    communityTemplateReadmeCollection:
```

Comment:
i told you to change it everywhere, but this is a place where it should be slightly different
```suggestion
    readme:
```
you'll have to change it elsewhere in the file, but i figured i'd keep from suggestion all of those